plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -2280,7 +2304,9 @@ def compare_radio_config(self, current_radio, want_radio):
                     unmatch_count = unmatch_count + 1
                     self.log("Antenna name unmatched: {0}".format(want_radio[dto_key]), "INFO")
                 elif dto_key == "cable_loss":
-                    temp_dtos[dto_key] = want_radio[dto_key]
+                    actual_gain = int(want_radio.get("antenna_gain", 0)) - int(want_radio[dto_key])
```

Comment:
Can we split it for clarity? Also you can log the values if required for debugging.

```
                    cable_loss = int(want_radio[dto_key])
                    antenna_gain = int(want_radio.get("antenna_gain", 0))
                    actual_gain = antenna_gain - cable_loss
                    
                    if current_radio.get(self.keymap["antenna_gain"]) != actual_gain:
                        temp_dtos[dto_key] = cable_loss
                        

```

Reply:
Updated